packages/pnpm-sync-lib/src/pnpmSyncPrepare.ts

```diff
@@ -58,17 +58,23 @@ export async function pnpmSyncPrepareAsync({
     }
   });
 
-  if (!fs.existsSync(lockfilePath)) {
-    throw Error('The input pnpm-lock.yaml path is not correct!');
+  if (!fs.existsSync(lockfilePath) || !fs.existsSync(storePath)) {
```

Comment:
should we use the `async` API in this `async` function?

Reply:
fs.exists API is deprecated.